roadmaps/android.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: contribute
+title: Android roadmap
+---
+
+# Android Roadmap
+
+*Last verified: 2018-07-21*
+([update history](https://github.com/bazelbuild/bazel-website/commits/master/roadmaps/android.md))
+
+**Points of contact:** [ahumesky](https://github.com/ahumesky), [jingwen](https://github.com/jin)
+
+## Overview
+
+Bazel is capable of building Android apps using the [Android
+rules](https://docs.bazel.build/versions/master/be/android.html). Blaze, the
+Google build tool that Bazel is open-sourced as, is used to build Google apps
```

Comment:
nit: "Blaze, Google's internal version of Bazel"

Reply:
done